src/cpp/user_interface/arguments_configuration.cpp

```diff
@@ -0,0 +1,291 @@
+// Copyright 2021 Proyectos y Sistemas de Mantenimiento SL (eProsima).
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @file arguments_configuration.cpp
+ *
+ */
+
+#include <vector>
+
+#if defined(_WIN32)
+#include  <stdlib.h>
+#else
+#include <unistd.h>
+#endif // if defined(_WIN32)
+
+#include <ddsrouter/user_interface/arguments_configuration.hpp>
+#include <ddsrouter/types/Log.hpp>
+
+namespace eprosima {
+namespace ddsrouter {
+namespace ui {
+
+const option::Descriptor usage[] = {
+    {
+        optionIndex::UNKNOWN_OPT,
+        0,
+        "",
+        "",
+        Arg::None,
+        "Usage: Fast DDS Router \n" \
+        "Connect different DDS networks via DDS through LAN or WAN.\n" \
+        "It will build a bridge between the different Participant configurations set.\n" \
+        "General options:"
+    },
+
+    {
+        optionIndex::HELP,
+        0,
+        "h",
+        "help",
+        Arg::None,
+        "  -h \t--help\t  \t" \
+        "Produce this help message."
+    },
+
+    {
+        optionIndex::CONFIGURATION_FILE,
+        0,
+        "f",
+        "config-path",
+        Arg::Readable_File,
+        "  -f \t--config-path\t  \t" \
+        "Path to the Configuration File (yaml format) [Default: ./DDS_ROUTER_CONFIGURATION.yaml]."
+    },
+
+    {
+        optionIndex::RELOAD_TIME,
+        0,
+        "r",
+        "reload",
+        Arg::Numeric,
+        "  -r \t--reload\t  \t" \
+        "Time period in miliseconds to reload configuration file. " \
+        "This is needed when FileWatcher functionality is not available (config file is a symbolic link). " \
```

Comment:
```suggestion
        "This is needed when FileWatcher functionality is not available (e.g. config file is a symbolic link). " \
```

Reply:
:+1:

Reply:
You wrote "i.e.", but I think "e.g." is better as there might be more reasons.

Reply:
:+1: